### PR TITLE
Show releases count in releases tab and persist it

### PR DESCRIPTION
**Pull request by hkdobrev**
Follow-up to https://github.com/sindresorhus/refined-github/pull/12#issuecomment-185583858

I made some small fixes/adjustments in separate commits which I needed for this PR.

### CODE REVIEW

**Comment by hkdobrev** [comment]
I'm appending the releases count after the releases tab is in the DOM and this can flash sometimes as the `chrome.storage` API is asynchronuous.

File: extension/content.js [hunk 1]
```diff
@@ -154,6 +162,16 @@ function addReleasesTab() {
...
+
+		if (isRepoRoot()) {
+			const releasesCount = $('.numbers-summary a[href$="/releases"] .num').text().trim();
+			appendReleasesCount(releasesCount);
+			chrome.storage.local.set({[releasesCountCacheKey]: releasesCount});
+		} else {
+			chrome.storage.local.get(releasesCountCacheKey, items => {
+				appendReleasesCount(items[releasesCountCacheKey]);
+			});
+		}
```
Comment (sindresorhus) [inline extension/content.js]: Maybe put all this in a function and call it from here for clarity. `cacheReleasesCount()`.

Comment (hkdobrev) [inline extension/content.js]: :white_check_mark:

**Comment by sindresorhus** [comment]
Looks good :)

**Pull request merged**